oblib/validator.py

```diff
@@ -77,12 +77,29 @@ def xbrli_boolean_item_type_validator(value, concept):
     """XBRLI boolean validator."""
     errors = []
     if type(value) is str:
-        if value.lower() not in ['true', 'false']:
+        if re.search(r'\d', value):
```

Comment:
Revert the changes to `xbrli_boolean_item_type_validator`, we're adding this capability in @michael-bbs 's branch.